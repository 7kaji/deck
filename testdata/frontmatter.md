---
title: YAML Frontmatter Test
author: Test Author
date: 2024-01-01
tags:
  - test
  - frontmatter
  - yaml
custom:
  nested: value
  number: 42
---

# Title with Frontmatter

## Subtitle

This is a test slide with YAML frontmatter.

<!-- {"layout":"title"} -->

---

# Second Slide

- Item 1
- Item 2
- Item 3

The frontmatter should only be available in the first slide.