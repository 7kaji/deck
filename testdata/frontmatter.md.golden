[
  {
    "layout": "title",
    "titles": [
      "Title with Frontmatter"
    ],
    "subtitles": [
      "Subtitle"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a test slide with YAML frontmatter."
              }
            ]
          }
        ]
      }
    ]
  },
  {
    "layout": "",
    "titles": [
      "Second Slide"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Item 1"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Item 2"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Item 3"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "The frontmatter should only be available in the first slide."
              }
            ]
          }
        ]
      }
    ]
  }
]